backend/substrapp/tests/generate_assets.py

```diff
@@ -28,6 +28,254 @@ def main(network):
     assets['traintuple'] = json.dumps(client.list_traintuple(), indent=4)
     assets['testtuple'] = json.dumps(client.list_testtuple(), indent=4)
     assets['computeplan'] = json.dumps(client.list_compute_plan(), indent=4)
+    assets['compositetraintuple'] = """[
```

Comment:
Can you explain why the composite traintuple asset is hard-coded ?

Reply:
The easiest way I found to make the unit tests pass (make it work, make it better). 
We are not using populate to make e2e test but substra-tests on this mission.
We should refactor unit tests, to be less dependent on fixtures, until that, I'm open to other solutions with your help.

Reply:
Solved by @jmorel work !